models/user.js

```diff
@@ -39,121 +41,109 @@ FROM User
 WHERE US_Username = ?
 LIMIT 1
 ;`;
-	var sanitsedInputs = [username];
-	db.query(query, sanitsedInputs, 
-		(err, results) => 
-		{ 
-			if (err) console.log("User.js | getUser | ERROR: " + err.message);
-			if (results.length > 0)
-			{
-				callback.found(convertToUserObject(results[0]));
-			}
-			else
-			{
-				callback.notFound();
-			}
-		});
+    var sanitsedInputs = [username];
+    db.query(query, sanitsedInputs,
+        (err, results) => {
+            if (err) console.log("User.js | getUser | ERROR: " + err.message);
+            if (results.length > 0) {
+                callback.found(convertToUserObject(results[0]));
+            } else {
+                callback.notFound();
+            }
+        });
 
 
-}
+};
 
 /**
  * Checks that a user exists according to the username provided
  * @param  {string} username - Username of user.
  * @param {userObject} callback - found() and notFound() expected
  */
-exports.checkUserExists = function(user, callback = {found: () => {}, notFound: () => {}})
-{
-	var db = database.connectDatabase();
-	var query = `
+exports.checkUserExists = function (username, callback = {
```

Comment:
leave as it was pls
or alternatively like this:
found: (user) => { },
notFound: () => { }